hash_test.go

```diff
@@ -117,6 +117,54 @@ func TestHash_BinaryMarshaler(t *testing.T) {
 	}
 }
 
+func TesHash_BinaryAppender(t *testing.T) {
```

Comment:
This func won't quite be run as a test yet: 😄 

```suggestion
func TestHash_BinaryAppender(t *testing.T) {
```